src/db/schema.ts

```
@@ -158,7 +158,7 @@ import {
   primaryKey,
   index,
 } from "drizzle-orm/pg-core";
-import { is } from "drizzle-orm";
+import { is, not } from "drizzle-orm";
```

ESLint

[error] Expected "drizzle-orm" to come before "drizzle-orm/pg-core".

(perfectionist/sort-imports)